Materials/IMaterials/StandardMaterials/EN/EnExecutionClass.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace MagmaWorks.Taxonomy.Materials.StandardMaterials.EN
+{
+    public enum EnExecutionClass
```

Comment:
@kpne I know you you made all ENThings not camelcase, but I think this has a lot better readability!